src/ui/profile_preferences.ui

```diff
@@ -726,6 +714,18 @@
             </property>
            </widget>
           </item>
+          <item row="2" column="0" colspan="4">
```

Comment:
Watch out for this reordering of `QGridLayout` `<item ...>`s - as I've remarked before it makes for Git noise - the  only fix seems to be hand-editing the `.ui` file in the plain-text editor - at least the code/segment folding makes it easy to collapse each `<item ...>` element down to a single line which can be cut-and-pasted back into the right spot - though beware that this gets messed up if you paste after the last (remaining) item or possibly before the first {so a careful pasting a new *last* item *before* the last *actual* one and then cutting the latter to reinsert it before the new last one is the convoluted way to do that} - which does become second nature if you have been working around this Qt Designer issue as long/often as *I* have...!

Reply:
I don't want to be manually re-order things by hand - it's OK as it is.

Reply:
Not really - as it makes it more likely to clash with any other PR that touches the `.ui` file - and makes the Git changes larger (a whole multi-line section representing an `<item ...> ... </item>` gets moved around {a deletion and an addition of largely identical XML} as well as an addition of the proper new part, compared to the insertion of a just new part {just an addition}). :frowning_face:

Reply:
I'm still not into forcing everybody to manually hand adjust XML. The benefits are not that high: merging UI files even then is still a pain.